OCD-3996: Stop accepting measures at the criteria level during upload

kekey1 opened the pull request.

kekey1 committed:
feat: Remove onetime job to update measures

I think we were keeping the job around to possibly make small updates to
and use again in the future, but it had a lot of code to deal with
legacy measure mappings and was using the legacy measure names. My
thought is that if we need to have a job remove measures again it will
be very different from this one which is why I decided to delete it
instead of update it.

[#OCD-3996]

kekey1 committed:
feat!: Stop accepting measures at the criteria level during upload

[#OCD-3996]

kekey1 committed:
feat: Remove macra -> Mips measure mappings for legacy measures

We were keeping the legacy measure usage around only for parsing
measures at the criteria level during listing upload.

[#OCD-3996]

kekey1 committed:
fix: Handle null in boolean comparison

[#OCD-3996]

kekey1 committed:
refactor: Remove unused reviewer

[#OCD-3996]

andlar reviewed (dismissed):
It's so fun seeing a PR that's 99% "delete a lot of code"!

kekey1 committed:
Merge branch 'staging' into OCD-3996

kekey1 committed:
Merge branch 'staging' into OCD-3996

kekey1 committed:
Merge branch 'staging' into OCD-3996

kekey1 committed:
Merge branch 'staging' into OCD-3996

kekey1 committed:
Merge branch 'staging' into OCD-3996

kekey1 committed:
fix: Use grammatically correct list of criteria in measure removal err msg

[#OCD-3996]

kekey1 committed:
fix: Re-instate combining of measures to avoid duplicates

Combining measures in the MeasureNormalizer allows measures to be
specified in multiple lines in the upload file but then combined into
one measure + associated criteria during normalization. This was
existing behavior (sort of a side effect of combing measures at the
criteria level with measures at the listing level) but it doesn't make
sense to remove that existing functionality for users.

[#OCD-3996]
